### ISSUE TITLE
Generated .obj not compatible with Godot 3.3.2 or Blender 2.83

### ISSUE THREAD

**Issue by belzecue** [fractilegames/godot-obj-export#1]
Godot 3.3.2 and Blender 2.83 can't import the OBJ files I generated from this tool.  Both see the files as corrupt.

![Screenshot at 2021-06-06 03-00-15](https://user-images.githubusercontent.com/1931303/120902539-89c72300-c673-11eb-8359-ec7e368f542a.png)

Here are some sample generated .obj files: https://c.mail.com/@976023017429673037/wkf4EbC-Q4SobG1VZlVITw

Thanks.

**Comment by fractilegames** [comment]
Found it. This is an embarrassing mistake. I must have broken this in my last minute refactoring. Thanks for letting me know!

**Comment by belzecue** [comment]
Just a final thank you for the fix.  You really saved my bacon.  Here's what happened.  I wanted to generate in Godot every mesh variation of a cube for all permutations of the six sides variously included or omitted, so 64 variations (inc full cube and empty cube, and ignoring duplicates that could be excluded via rotation).  That part was easy.  Next, I needed to get these individual cube instances merged for each visible face.  That too was easy, thanks to "surface tool. append from."  The hard part?  Needing to export these generated meshes out of Godot.  There is literally no way to do it without your script.  It's a frustrating situation that everyone seems to concentrate on getting meshes into Godot (chiefly from Blender) and nobody thinks about getting them out -- as if the idea of anyone doing procgen meshes in Godot is preposterous.  In several hours of Googling I am convinced nobody solved this except you and the dude who made the CSGObjExport script -- which I had already begun to rewrite for my purposes, but it was going to be a long slog.

Anyway, I slotted in your change just now and voila!  My 63 meshes now freshly minted and ready to import into Blender.  Thank you so much.
